containers/full

```diff
@@ -0,0 +1,24 @@
+FROM registry.fedoraproject.org/fedora
+
+# Adjust system defaults
+RUN set -x && \
+    sed -i '/tsflags=nodocs/d' /etc/dnf/dnf.conf && \
+    echo "Set disable_coredump false" >> /etc/sudo.conf && \
+    echo 'tester ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+
+# Install necessary packages
+RUN set -x && \
+    dnf install -y tmt-all beakerlib python3-{mock,test} && \
+    dnf autoremove -y && \
+    dnf clean all
+
+# Prepare files for experimenting
+RUN useradd --uid 1001 --create-home tester
+COPY ../.fmf /home/tester/.fmf
+COPY ../tests /home/tester/tests
```

Comment:
I don't see how copying `tmt` tests could be benefitial for someone. Could you elaborate on the expected use?

Reply:
That's why I suggested to clarify the use case: I understand this as a nice & safe way how to start experimenting with `tmt`. And for such experimenting it's useful to have tests and plans ready for playing.

Reply:
Sure, for the `full` container it's fine with me :). (Also it doesn't inhibit the advanced usage.)
For the `mini` container, it depends on the particular use. But that's an RFE, not a blocker.

_ _ _ _

BTW, a standard way to save various dockerfiles is to add suffix to the `Dockerfile` name.

E.g. `Dockerfile.full`.

Reply:
Makes sense, renamed.